Source/WebKit/Configurations/BaseTarget.xcconfig

```diff
@@ -141,6 +74,7 @@ WK_SKIP_INSTALL_WEBPUSHD[sdk=macosx*] = $(WK_OR_$(WK_RELOCATABLE_FRAMEWORKS)_$(W
 WK_SKIP_INSTALL_WEBPUSHD[sdk=macosx11*] = YES;
 WK_SKIP_INSTALL_WEBPUSHD[sdk=macosx12*] = YES;
 
+// FIXME: There is no need for a variable that's always YES. If this ever needs to be disabled, it should be a USE, not a HAVE.
 WK_HAVE_CORE_PREDICTION = YES;
```

Comment:
Was curious about this setting so I looked up how it's used. As a follow up, I think we can simplify it to just this line in WebKit.xcconfig:

```
WK_CORE_PREDICTION_LDFLAGS = $(WK_MACOS_WEAK_FRAMEWORK) CorePrediction
```

and we'll need to unifdef the six `HAVE(CORE_PREDICTION)` blocks in WebKit.